Jellyfin.Plugin.Webhook/Destinations/GenericForm/GenericFormOption.cs

```diff
@@ -14,6 +14,7 @@ public GenericFormOption()
     {
         Headers = Array.Empty<GenericFormOptionValue>();
         Fields = Array.Empty<GenericFormOptionValue>();
+        // The MediaType is left as JSON because that is what is assumed to be used in GenericFormClient.cs
```

Comment:
Json is not assuimed in GenericFormClient, the content type should be `application/x-www-form-urlencoded`

Reply:
It may not be _intended_ to be, but the body is currently treated solely as JSON per current master: https://github.com/jellyfin/jellyfin-plugin-webhook/blob/c507d/Jellyfin.Plugin.Webhook/Destinations/GenericForm/GenericFormClient.cs#L52-L58 
Once in the processing itself the Generic Client does use FormURLContent (line 77 of same file, unchanged in PR)